config.json

```diff
@@ -1,4 +1,4 @@
 {
-    "CAIRO_LANG": "0.8.1",
-    "STARKNET_DEVNET": "0.2.0"
+    "CAIRO_LANG": "0.8.2",
```

Comment:
What about `0.8.2.1`?

Reply:
totally forgot about that

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/111/commits/e9cbce03eb3528eee7c8d7d7ed6ae01a40385215